packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -44,12 +41,35 @@ module.exports = {
               ],
             },
           },
+          overrideBrowserslist: {
+            oneOf: [{ type: 'string' }, { type: 'array', items: { type: 'string' } }],
+          },
+          browserslistOptions: {
+            type: 'object',
+            properties: {
```

Comment:
Why constrain to these particular ones? It's not the full set of current ones.

On balance I'd prefer allowing everything through to be compatible with whatever browserlist might add/remove in future.

Reply:
Can I define the option as any somehow then ?

Reply:
2ab8069921bd69fbc73282890975b3a051e430ca

Reply:
Thanks for resolving these last issues